Packs/ExpanseV2/Scripts/ExpanseAggregateAttributionCI/ExpanseAggregateAttributionCI.py

```diff
@@ -0,0 +1,101 @@
+from CommonServerUserPython import *  # noqa
+
+from typing import Dict, Any, Tuple, Optional
+import traceback
+
+
+''' STANDALONE FUNCTION '''
+
+
+def deconstruct_entry(ServiceNowCMDBContext: Dict[str, Any]) -> Tuple[Optional[str],
+                                                                      Optional[str],
+                                                                      str,
+                                                                      Optional[str],
+                                                                      Optional[str],
+                                                                      Optional[str]]:
+    """
+    deconstruct_entry
+    Extracts device relevant fields from a log entry.
+
+    :type ServiceNowCMDBContext: ``Dict[str, Any]``
+    :param ServiceNowCMDBContext: ServiceNowCMDB.Record
+
+    :return: Tuple where the first element is the name or None, the second element is the
+        sys class name or None, the third element is the sys_id or None, the fourth element is the
+        asset display value or None, the fifth element is the asset link or None, and the final element
+        is the asset value or None.
+    :rtype: ``Tuple[Optional[str], Optional[str], Optional[str], Optional[int], Optional[str], Optional[str]]``
+    """
+    name = ServiceNowCMDBContext.get("name")
+    sys_class_name = ServiceNowCMDBContext.get("sys_class_name")
+    sys_id = ServiceNowCMDBContext.get("sys_id", "Unknown")
+    asset_display_value = ServiceNowCMDBContext.get("asset", {}).get("display_value")
+    asset_link = ServiceNowCMDBContext.get("asset", {}).get("link")
+    asset_value = ServiceNowCMDBContext.get("asset", {}).get("value")
+
+    return name, sys_class_name, sys_id, asset_display_value, asset_link, asset_value
+
+
+''' COMMAND FUNCTION '''
+
+
+def aggregate_command(args: Dict[str, Any]) -> CommandResults:
+    input_list = argToList(args.get('input', []))
+    current_list = argToList(args.get('current', []))
+
+    current_sys_ids = {
+        f"{c.get('sys_id', 'Unknown')}": c
+        for c in current_list if c is not None
+    }
+
+    for entry in input_list:
+        if not isinstance(entry, dict):
+            continue
+
+        name, sys_class_name, sys_id, asset_display_value, asset_link, asset_value = deconstruct_entry(entry)
+
+        current_state = current_sys_ids.get(sys_id)
+        if current_state is None:
+            current_state = {
+                'name': name,
+                'sys_id': sys_id,
+                'sys_class_name': sys_class_name,
+                'asset_display_value': asset_display_value,
+                'asset_link': asset_link,
+                'asset_value': asset_value,
+            }
+            current_sys_ids[sys_id] = current_state
+
+    markdown = '## ExpanseAggregateAttributionCI'
+    outputs = list(current_sys_ids.values())
+
+    return CommandResults(
+        readable_output=markdown,
```

Comment:
Is this by design that the readable_output does not include the data itself but only the header?